.github/workflows/test.yml

```diff
@@ -21,14 +21,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [windows-latest]
 
     steps:
     - uses: actions/checkout@v2
+    - run: |
+        npm install
+        npm run all
     - uses: ./
+    - name: Run conda list
+      run: conda list
+    - name: init pwsh
+      run: conda init powershell
```

Comment:
I *think* `conda init powershell` only initialises the *old* powershell (v5) and not he new `pwsh`. Worth a try at least.

See also: https://github.com/TheSnakePit/setup-mamba/issues/4#issuecomment-643137699